test/src/test/testtool/FxF3/common.properties

```diff
@@ -84,3 +84,6 @@ ignoreContentBetweenKeys15.key2=)
 
 ignoreContentBetweenKeys16.key1=<fielddefinition>
 ignoreContentBetweenKeys16.key2=</fielddefinition>
+
+ignoreContentBetweenKeys17.key1=fSize="
+ignoreContentBetweenKeys17.key2="
```

Comment:
This doesnt look right

Reply:
I know right! I can assure you that this is the way.

Reply:
But how does it work? Normally everything betwen `" "` should be a string right?
And how does `fSize="` work? or is it supposed to be a string by itself?

Reply:
It takes two strings, key1 the start, and key2 the end, it replaces the content-between-keys.